### PR TITLE
add mc tweak disableHoneyBlockEffect

### PR DESCRIPTION
**Pull request by topi-banana**

### CODE REVIEW

File: src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mc_tweaks/disableHoneyBlockEffect/HoneyBlockMixin.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import static me.fallenbreath.tweakermore.util.ModIds.minecraft;
+
+@Restriction(require = @Condition(value = minecraft, versionPredicates = ">=1.15"))
+@Mixin(HoneyBlock.class)
+public abstract class HoneyBlockMixin
+{
+  @Inject(method = "onEntityCollision", at = @At("HEAD"), cancellable = true)
```
Comment (Fallen-Breath) [inline src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mc_tweaks/disableHoneyBlockEffect/HoneyBlockMixin.java:43]: Suggestion: apply `@ModifiyExpressionValue` onto the `isSliding` call. Much better than inject + cancel

**Review by Fallen-Breath — CHANGES_REQUESTED** [review]
- Please keep the coding style, i.e. tab + brace-on-new-line on all files
- Please maintain the config name / description in the language file (at least for en_us)

**Comment by topi-banana** [comment]
thank you for your review.
I've made the necessary changes.

File: src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mc_tweaks/disableHoneyBlockEffect/HoneyBlockMixin.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.entity.Entity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+@Restriction(require = @Condition(value = ModIds.minecraft, versionPredicates = ">=1.15"))
+@Mixin(HoneyBlock.class)
+public abstract class HoneyBlockMixin
+{
+  @ModifyExpressionValue(
```
Comment (Fallen-Breath) [inline src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mc_tweaks/disableHoneyBlockEffect/HoneyBlockMixin.java:39]: code styling

Comment (topi-banana) [reply src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mc_tweaks/disableHoneyBlockEffect/HoneyBlockMixin.java:39]: Uh, indent space -> tab?

**Comment by topi-banana** [comment]
Is this okay?

**Review by Fallen-Breath — APPROVED** [review]
LGTM

**Pull request merged**